book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_acessor :title, :author, :rentals
```

Comment:
- [ ] Please fix this typo. `attr_accessor` is the correct spelling. You only used one `c`